### PR TITLE
Renaming AST to ONE

### PR DESCRIPTION
**Pull request by igorlombacx**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

> Describe the purpose of this PR along with any background information and the impacts of the proposed change.

### References

https://checkmarx.atlassian.net/browse/AST-18921?atlOrigin=eyJpIjoiZGMxMjVmZjVjMTg4NDU0M2FmODEyNjljMWUwMGNlNmEiLCJwIjoiaiJ9

### Testing

> Describe how this change was tested. Be specific about anything not tested and reasons why. If this solution has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

**Review by tiagobcx — APPROVED** [review]

File: internal/commands/auth.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 const (
 	failedCreatingClient = "failed creating client"
 	pleaseProvideFlag    = "%s: Please provide %s flag"
-	SuccessAuthValidate  = "Successfully authenticated to AST server!"
+	SuccessAuthValidate  = "Successfully authenticated to ONE server!"
```
Comment (pedrompflopes) [inline internal/commands/auth.go]: Lets add checkmarx One here instead of just one.

File: internal/commands/predicates.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ func NewResultsPredicatesCommand(resultsPredicatesWrapper wrappers.ResultsPredic
 	triageCmd := &cobra.Command{
 		Use:   "triage",
 		Short: "Manage results",
-		Long:  "The 'triage' command enables the ability to manage results in CxAST.",
+		Long:  "The 'triage' command enables the ability to manage results in CxONE.",
```
Comment (pedrompflopes) [inline internal/commands/predicates.go]: Lets rename to checkmarx one as well

File: internal/commands/root.go [hunk 3]
```diff
@@ -46,8 +46,8 @@ func NewAstCLI(
 	// Create the root
 	rootCmd := &cobra.Command{
 		Use:   "cx <command> <subcommand> [flags]",
-		Short: "Checkmarx AST CLI",
-		Long:  "The AST CLI is a fully functional Command Line Interface (CLI) that interacts with the Checkmarx CxAST server.",
+		Short: "Checkmarx One CLI",
+		Long:  "The One CLI is a fully functional Command Line Interface (CLI) that interacts with the Checkmarx CxOne server.",
```
Comment (pedrompflopes) [inline internal/commands/root.go:50]: just checkmark One here.

File: internal/wrappers/configuration/configuration.go [hunk 4]
```diff
@@ -145,7 +145,7 @@ func ShowConfiguration() {
 	fmt.Println(viper.GetString(params.BaseURIKey))
 	fmt.Printf("%30v", "BaseAuthURIKey: ")
 	fmt.Println(viper.GetString(params.BaseAuthURIKey))
-	fmt.Printf("%30v", "AST Tenant: ")
+	fmt.Printf("%30v", "ONE Tenant: ")
```
Comment (pedrompflopes) [inline internal/wrappers/configuration/configuration.go]: Checkmarx One here.

**Review by pedrompflopes — APPROVED** [review]

**Pull request merged**